### PR TITLE
Update Inspector issues for single-source-issues

### PR DESCRIPTION
**Pull request by hugihlynsson**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/inspector/_meta**](https://docs-qzyngbc74.avo-dev.app/inspector/_meta) | [**…/inspector-issues-beta**](https://docs-qzyngbc74.avo-dev.app/inspector/inspector-issues-beta) |
|:-:|:-:|
|<a href="https://docs-qzyngbc74.avo-dev.app/inspector/_meta">Screenshot of /inspector/_meta</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/-54goCyWRBfpA2lmQgtENkR_3GzjUABD_vtP0rCu3JhrszgKlSUxTVOggkjknwPeZgKVTd1LzeUPPvxHjdRp1A.png">(view full size)</a> |<a href="https://docs-qzyngbc74.avo-dev.app/inspector/inspector-issues-beta">Screenshot of /inspector/inspector-issues-beta</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/AGWJRTTsNHbWs0u9DFK2_aSiSDWM3PyExjteZrF9mBMi-SHFAZM9WYYJbub-YGRzguSu4FUxpDYmLNUIWtXC8w.png">(view full size)</a> |



| [**…/inspector-issues-view**](https://docs-qzyngbc74.avo-dev.app/inspector/inspector-issues-view) |
|:-:|
|<a href="https://docs-qzyngbc74.avo-dev.app/inspector/inspector-issues-view">Screenshot of /inspector/inspector-issues-view</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/j-ZRscFKQvh0AZjQOwN8lf4W6GZV4g91wlzEor3ZHeSNDiOtJ8qG9s3RBSsWDT15a10ibbwjNrNKoqVZVCyrQQ.png">(view full size)</a> |







Commit e39eddaeef42951391f0bd508ee460fd85d27e09 (https://docs-qzyngbc74.avo-dev.app).

File: pages/inspector/inspector-issues-view.mdx [hunk 1]
```diff
@@ -24,14 +27,13 @@ To narrow down your search to your most relevant issues, you can filter by the f
...
 - Event name
 - Property
-– Category
-– Tag
+- Category
+- Tag
 
 ![Image shows the inspector Issues table ordered by issue volume and filtered by Web source.](/images/inspector/inspector-issues-filtering.png)
 
 > 💡 We are actively working on adding more filter options, and you can expect more filters in the near future. If there’s anything specific you’d like to be able to filter by, please [let us know](/help/troubleshooting).
```
Comment (bragur) [inline pages/inspector/inspector-issues-view.mdx:35]: Use Callout here?

**Review by bragur — APPROVED** [review]
LGTM!

File: pages/inspector/inspector-issues-view.mdx (hunk 1 above)
Comment (hugihlynsson) [reply pages/inspector/inspector-issues-view.mdx:35]: Yes!

Comment (hugihlynsson) [reply pages/inspector/inspector-issues-view.mdx]: Yes!

**Pull request merged**